docs/content/en/docs/intro-klt.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Introduction to the Keptn Lifecycle Toolkit
+linktitle: Introduction to the Keptn Lifecycle Toolkit
+description: Understand the Keptn Lifecycle Toolkit
+weight: 05
+cascade:
+  github_subdir: "docs/content/en/docs"
+  path_base_for_github_subdir: "/content/en/docs-dev"
+---
+
+The Keptn Lifecycle Toolkit (KLT)  makes your deployments observable,
+brings application-awareness to your Kubernetes cluster,
+and helps you reliably deliver your application with:
+
+* Pre-Deployment Tasks: e.g. checking for dependant services,
+  checking if the cluster is ready for the deployment, etc.
```

Comment:
I think we should suggest also that tasks are for changing things
```suggestion
  setting the cluster to be ready for the deployment, etc.
```

Reply:
done